### PR TITLE
ParseCommandLine: Add Command for setting Volume

### PR DESCRIPTION
**Pull request by precla**
Feature request trac: #5991
Add 'volume' switch to launch File with desired volume level.

### CODE REVIEW

File: src/mpc-hc/MainFrm.cpp [hunk 1]
```diff
@@ -11189,6 +11189,10 @@ void CMainFrame::OpenSetupAudio()
 {
     m_pBA->put_Volume(m_wndToolBar.Volume);
 
+    if (AfxGetAppSettings().nCLSwitches & CLSW_VOLUME) {
```
Comment (kasper93) [inline src/mpc-hc/MainFrm.cpp]: Why this is needed? You already set volume in setting...

File: src/mpc-hc/AppSettings.cpp [hunk 2]
```diff
@@ -2105,6 +2105,12 @@ void CAppSettings::ParseCommandLine(CAtlList<CString>& cmdln)
                 nCLSwitches |= CLSW_ADD;
             } else if (sw == _T("randomize")) {
                 nCLSwitches |= CLSW_RANDOMIZE;
+            } else if (sw == _T("volume")) {
```
Comment (precla) [inline src/mpc-hc/AppSettings.cpp]: Added

File: src/mpc-hc/MainFrm.cpp (hunk 1 above)
Comment (precla) [inline src/mpc-hc/MainFrm.cpp]: Without it, it doesn't set the Volume at all. Only on 2nd launch of mpc-hc. I don't know why and what I did wrong

Comment (matthewmauger) [reply src/mpc-hc/MainFrm.cpp:11192]: Possible race condition due to that line getting hit from the graph thread before the toolbar volume has initialised?

**Comment by kasper93** [comment]
Thanks!

**Pull request closed without merging**